perception/autoware_tensorrt_yolox/src/tensorrt_yolox.cpp

```diff
@@ -156,16 +156,25 @@ namespace autoware::tensorrt_yolox
 TrtYoloX::TrtYoloX(
   const std::string & model_path, const std::string & precision, const int num_class,
   const float score_threshold, const float nms_threshold, tensorrt_common::BuildConfig build_config,
-  const bool use_gpu_preprocess, std::string calibration_image_list_path, const double norm_factor,
-  [[maybe_unused]] const std::string & cache_dir, const tensorrt_common::BatchConfig & batch_config,
-  const size_t max_workspace_size, const std::string & color_map_path)
+  const bool use_gpu_preprocess, const int gpu_id, std::string calibration_image_list_path,
+  const double norm_factor, [[maybe_unused]] const std::string & cache_dir,
+  const tensorrt_common::BatchConfig & batch_config, const size_t max_workspace_size,
+  const std::string & color_map_path)
 {
+  gpu_id_ = gpu_id;
+  std::cout << "GPU" << gpu_id_ << "is selected for the inference !" << std::endl;
```

Comment:
@ismetatabay 

Please use `RCLCPP_INFO` or `RCLCPP_INFO_THROTTLE`.
cf. https://github.com/autowarefoundation/autoware.universe/blob/3551d649d390ec87daa57695742985c5370ebc93/perception/autoware_tensorrt_yolox/src/tensorrt_yolox_node.cpp#L109

Reply:
Hello @Shin-kyoto -san, thank you for your valuable suggestions. We've moved our development node to the tensorrt_yolox library (```tensorrt_yolox.cpp```). I am not sure whether to pass rclcpp into ```tensorrt_yolox.cpp``` for logging purposes. What do you think about that?